src/shared/getMetaInfo.js

```diff
@@ -27,5 +29,20 @@ export default function getMetaInfo (options = {}, info, escapeSequences = [], c
     info.base = Object.keys(info.base).length ? [info.base] : []
   }
 
+  if (info.meta) {
+    // remove meta items with duplicate vmid's
+    info.meta = info.meta.filter((metaItem, index, arr) => {
+      return (
+        // keep meta item if it doesnt has a vmid
+        !metaItem.hasOwnProperty(tagIDKeyName) ||
```

Comment:
maybe we could use named variables instead of comments here? e.g.

```js
const hasNoVmid = !metaItem.hasOwnProperty(tagIDKeyName)

if (hasNoVmid) {
 return true
}

const firstItemWithVmid = index === findIndex(arr, item => item[tagIDKeyName] === metaItem[tagIDKeyName]

return firstItemWithVmid
```

So the comments can be directly transformed into code and "can't lie".

Reply:
+1, probably for the better indeed. Although if the comments would lie then the variable name would lie probably as well as it would be the same steps of _creating new code_ and _changing code but not updating descriptions_

Could you also please have a look at the potential break I mentioned? Did you mean for it to be possible to recursively apply templates so you could get something like `My Component - My Page - My Layout - My Website`?

Reply:
>  Although if the comments would lie then the variable name would lie probably as well as it would be the same steps of creating new code and changing code but not updating descriptions

Partially agree! However, I personally would likely think sooner about changing a variable name then changing a comment.

> Did you mean for it to be possible to recursively apply templates

I actually never thought about that :see_no_evil: